group_vars/all/vars.yml

```diff
@@ -136,3 +136,16 @@ running_on_server: true
 postgresql_is_cloud: false
 postgresql_is_local: false
 slack_alerts_channel: '#ansible-alerts'
+ufw_campus_and_vpn: 
+  - 10.249.64.0/18 #Princeton Wired Private
+  - 10.249.0.0/18 #Princeton Wired Private-2
+  - 128.112.0.0/16 #Princeton Wired
+  - 172.20.80.0/22 #PU Subnet - LibNetPvt
+  - 172.20.95.0/24 #Princeton VPN Subnet 1
+  - 172.20.192.0/19 #Princeton VPN Subnet 2
+ufw_libnet: 
```

Comment:
We should come up with a new name?
- ufw_public_library_servers
- or, document what libnet means